### ISSUE TITLE
reduce_mean axis error

### ISSUE THREAD

**Issue by ChenMinQi** [jyhengcoder/Gradient-Centralization_tensorflow#1]
Since the shape of conv weights is K*K*C_in*C_out，so it maybe

`grad -= tf.reduce_mean(grad, axis=list(range(len(grad.shape)-1), keep_dim=True)`
